### PR TITLE
CNV BZ#1874084 removing unsupported NICs

### PR DESCRIPTION
**Pull request by aburdenthehand**
Removing the unsupported NIC types

https://bugzilla.redhat.com/show_bug.cgi?id=1874084
@phoracek

### CODE REVIEW

**Comment by aburdenthehand** [comment]
Removing enterprise-4.5 label as the original UI bug is targetted to OCP 4.6

File: modules/virt-networking-wizard-fields-web.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 |Name for the Network Interface Card.
 
 |Model
-|Indicates the model of the Network Interface Card. Supported values are *e1000*, *e1000e*, *ne2k_pci*, *pcnet*, *rtl8139*, and *virtIO*.
+|Indicates the model of the Network Interface Card. Supported values are *e1000e* and *virtIO*.
```
Comment (yossisegev) [inline modules/virt-networking-wizard-fields-web.adoc:19]: The way we indicate the virtio model in the specs is "virtio" (all non-capitals), not "virtIO".
It's not even written this way in the formal documentation (e.g. https://www.redhat.com/en/blog/introduction-virtio-networking-and-vhost-net, https://wiki.libvirt.org/page/Virtio), even when referring to the model name itself (let alone when using it as a spec value).

**Review by yossisegev — CHANGES_REQUESTED** [review]

File: modules/virt-networking-wizard-fields-web.adoc (hunk 1 above)
Comment (aburdenthehand) [reply modules/virt-networking-wizard-fields-web.adoc]: Updated to match UI

**Pull request merged**